features/step_definitions/common_steps.rb

```diff
@@ -210,3 +210,7 @@
   overwrite_constant :ENABLED, false, Rack::NoPopups
 end
 
+Given(/^the main hearing date feature is enabled$/) do
+  ENV['MAIN_HEARING_DATE_ENABLED'] = 'true'
```

Comment:
i'm curious, will this revert back to the original value of MAIN_HEARING_DATE_ENABLED after the tests are complete?

Reply:
As it turns out, yes this is persistent. I have changed it so that it gets set at the start of the suite instead of individually on a feature.